website/src/parsers/yaml/yaml-unist-parser.js

```diff
@@ -0,0 +1,35 @@
+import defaultParserInterface from '../utils/defaultParserInterface';
+import pkg from 'yaml-unist-parser/package.json';
+
+const ID = 'yaml-unist-parser';
+
+export default {
+  ...defaultParserInterface,
+
+  id: ID,
+  displayName: ID,
+  version: pkg.version,
+  homepage: pkg.homepage || 'https://www.npmjs.com/package/yaml-unist-parser',
+
+  _ignoredProperties: new Set(['parent', 'errors', 'type']),
+  locationProps: new Set(['position']),
+
+  nodeToRange(node) {
+    if (node.position)
```

Comment:
We probably need to exclude `Null` node here since its position is always `-1`.

Reply:
is that when `node.type = "null"`?

Reply:
https://github.com/ikatyang/yaml-unist-parser/blob/13fd6fb4ccbb465399a165864bde1312a2461755/src/types.ts#L89-L93